hooks/pre/30-upgrade.rb

```diff
@@ -164,6 +171,7 @@ def fail_and_exit(message)
     upgrade_step :mark_qpid_cert_for_update
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :remove_gutterball
+    upgrade_step :delete_event_queue
     upgrade_step :start_tomcat, :run_always => true
```

Comment:
The risk in this case is that Candlepin will send events on the topic which has nobody listening. Any events sent during this time period are effectively being piped to /dev/null.